app/controllers/users/organisations_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Users::OrganisationsController < ApplicationController
+  skip_before_action :set_selected_organisation
+  skip_after_action :verify_policy_scoped
+
+  before_action :redirect_to_dashboard_if_cis2_is_enabled
+
+  layout "two_thirds"
+
+  def new
+    @organisations = current_user.organisations
+  end
+
+  def create
+    organisation = current_user.organisations.find(params[:organisation_id])
+
+    if organisation.present?
+      session["cis2_info"] = {
+        "selected_org" => {
+          "name" => organisation.name,
+          "code" => organisation.ods_code
+        },
+        "selected_role" => {
+          "code" => valid_cis2_roles.first
```

Comment:
I wonder if we should include the `workgroups` here? Just so it matches what we'll get from CIS2 and helps with local testing. In the future we could also add the option to sign in as an admin maybe as well?